87: [Data.RS] Exceptions if geometries are missing

stundzig opened the pull request:
@fb71 reicht es nicht vllt. aus, wenn ich den Test nach den _getBounds_, den Du sowieso in adaptMapBBOX drin hast, einfach in die NewLayerOperation reinziehe? Das hat zumindest bei meinem CSV und beim externen WMS jetzt funktioniert.

Die andere Methode mit dem connectLayer funktioniert nicht, weil ich aus der PipelineFeatureSource den FeatureType nicht rausbekomme. Dort müsste ich noch den GetFeatureTypeRequest implementieren, das würde ich aber gern vermeiden.

Commit:
87: [Data.RS] Exceptions if geometries are missing 

Task-Url: http://github.com/Polymap4/polymap4-core/issues/issue/87

  - add check is res has Bounds

fb71 commented:
Der ganze test ist überflüssig. Das entscheidende ist -!nativeLayerBounds.isNull()-. Wenn RsFeatureStore.getBounds() richtig regiert, was es jetzt tut, dann is ein "null" bounds legal, steht auch in der docu und alles ist gut.

Ich habe schon die entsprechenden änderungen gemacht. Ich suche nur gerade noch wo sonst noch solche komischen anwendungen von localFeatureStore() sind.

stundzig commented:
Ok, sehr gut. Dann dürfen wir adaptMapBBOX also nur aufrufen wenn -!nativeLayerBounds.isNull()-? Andernfalls kriegen wir ja sonst die Exception.

Commit:
87: [Data.RS] Exceptions if geometries are missing 

Task-Url: http://github.com/Polymap4/polymap4-core/issues/issue/87

  - clean up

Commit:
87: [Data.RS] Exceptions if geometries are missing 

Task-Url: http://github.com/Polymap4/polymap4-core/issues/issue/87

  - fixed layers visible check

stundzig commented:
@fb71 im layerspanel hab ich das auch auf res.getBounds() geändert. Das ist klarer in der Verwendung.

fb71 commented:
ich hab doch gesagt, dass ich dran bin. wenn du jetzt auch an den entsprechenden code-stellen hecktisch was änderst, dann hab ich blos einen haufen probleme beim mergen. Kannst du vielleicht einfach kurz warten?

stundzig commented:
Ok, dann warte ich.
